src/fides/api/task/execute_request_tasks.py

```diff
@@ -51,26 +54,29 @@ def run_prerequisite_task_checks(
             f"Privacy request with id {privacy_request_id} not found"
         )
 
+    if privacy_request.status == PrivacyRequestStatus.canceled:
+        raise PrivacyRequestStatusCanceled(
+            f"Cannot execute request task {privacy_request_task_id} of privacy request {privacy_request_id}: status is {privacy_request.status.value}"
+        )
+
```

Comment:
New check to exit a Request Task early if we realize the overall Privacy Request was cancelled.